test/plugin/duration.test.js

```diff
@@ -27,6 +27,11 @@ describe('Creating', () => {
     expect(dayjs.duration(60, 'seconds').toISOString()).toBe('PT1M')
     expect(dayjs.duration(13213, 'seconds').toISOString()).toBe('PT3H40M13S')
   })
+  it('two argument will bubble up to the next (negative number)', () => {
+    expect(dayjs.duration(-59, 'seconds').toISOString()).toBe('PT-59S')
+    expect(dayjs.duration(-60, 'seconds').toISOString()).toBe('PT-1M')
+    expect(dayjs.duration(-13213, 'seconds').toISOString()).toBe('PT-3H-40M-13S')
```

Comment:
seems this is not the correct result?
-PT59S
-PT1M
-PT3H40M13S

Reply:
Fixed